rpcs3/rpcs3qt/rsx_debugger.cpp

```diff
@@ -26,7 +26,7 @@ constexpr auto qstr = QString::fromStdString;
 namespace
 {
 	template <typename T>
-	std::span<T> as_const_span(std::span<const std::byte> unformated_span)
+	std::span<T> std::span(std::span<const std::byte> unformated_span)
```

Comment:
Error in function name.

Reply:
If you're replacing this function anyway, you can just remove it alltogether